packages/components/src/components/ScalarModal/ScalarModal.stories.ts

```diff
@@ -83,7 +83,8 @@ export const Search: Story = {
               <ScalarSearchResultItem
                 v-for="result in results"
                 :key="result.title"
-                icon="Search">
+                icon="Search"
+                href="#">
```

Comment:
non blocking/not coming from this but seems like we have a double modal in the storie